src/applications/vaos/containers/ExpressCareReasonPage.jsx

```diff
@@ -52,6 +53,18 @@ const uiSchema = {
   },
 };
 
+function recordNewAppointmentEvent() {
+  recordEvent({
+    event: `${GA_PREFIX}-express-care-new-appointment-clicked`,
```

Comment:
This event name sounds a little bit confusing since we're routing them to the regular flow.

Reply:
Yeah, I agree. I wanted to keep the `express-care` part to keep the EC events grouped. Maybe `vaos-express-care-switch-to-appointment-flow-clicked`?

Reply:
Yup that's definitely clearer